util/jsonhelp.go

```diff
@@ -0,0 +1,52 @@
+package util
+
```

Comment:
I'd add a comment somewhere in this file talking about how these methods are different...